wordpress-mobile/WordPress-iOS
# Sidebar: Foundation

This is the initial PR adding the minimum required infrastructure for supporting the sidebar on iPad.

- These changes are intended to **not** affect the iPhone version in any way
- The changes are under the new `.sidebar` feature flag

None of the UI/UX introduced in this PR is final. I'll continue working on it on the screen-by-screen basis in the future PRs with the two main goals:

- (User-facing) Replace a tab bar on iPad with a sidebar as these tab bars are no longer supported on iPadOS 18
- (Technical) Remove `WPSplitViewController` and all the associated complexity

I will avoid making any other changes for now.

## To test:

Since none of the change are final, the main priority for testing is to ensure the iPhone is un-affected by this change, and the iPad version continues to work as is when the FF is off.


https://github.com/user-attachments/assets/552eaeac-0cd6-46e7-bcd7-8dfb78ee31e7



## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


Testing checklist:
